dev/io.openliberty.mcp.internal_fat/fat/src/io/openliberty/mcp/internal/fat/tool/asyncToolApp/AsyncLifecycleTools.java

```diff
@@ -51,8 +51,6 @@ public CompletionStage<String> asyncLifecycleEcho(@ToolArg(name = "input", descr
             LOG.info("[LOGGED] AsyncLifecycleTools.asyncLifecycleEcho Tool throwing error");
             throw new RuntimeException("Method call caused runtime exception. This is expected if the input was 'throw error'");
         }
-        return executor.supplyAsync(() -> {
-            return input + ": (async)";
-        });
```

Comment:
These are actually two different cases - in one the `CompletionStage` is complete when it's returned and in the other it isn't. We actually want to test both cases.